lib/onelogin/ruby-saml/authrequest.rb

```diff
@@ -28,6 +28,8 @@ def create(settings, params = {})
 
       def create_params(settings, params={})
         params = {} if params.nil?
+        # Some ruby-saml versions uses :RelayState others use 'RelayState'
```

Comment:
True, perhaps we should be more explicit in the documentation of this method then.  Also, the assignment of `params = {} if params.nil?` seems unnecessary on line 30.